libsrc/Makefile

```diff
@@ -76,8 +78,11 @@ all lib: $(TARGETS)
 mostlyclean:
 	$(call RMDIR,../libwrk)
 
+# Transitional line active. Final line commented out below in order to
+# allow some time for transition between the directory structures
 clean:
-	$(call RMDIR,../libwrk ../lib ../targetutil $(addprefix ../,$(DRVTYPES)))
+	$(call RMDIR,../libwrk ../lib ../targetutil ../target $(addprefix ../,$(DRVTYPES)))
+# 	$(call RMDIR,../libwrk ../lib ../target)
```

Comment:
I don't think that such a transition is necessary / desirable. Please remove it.